gerrit-entrypoint.sh

```diff
@@ -168,7 +168,13 @@ if [ "$1" = "/gerrit-start.sh" ]; then
   [ -z "${HTTPD_LISTENURL}" ] || set_gerrit_config httpd.listenUrl "${HTTPD_LISTENURL}"
 
   #Section gitweb
-  set_gerrit_config gitweb.cgi "/usr/share/gitweb/gitweb.cgi"
+  if [ "${GITWEB_TYPE}" = 'gitiles' ]  ; then
+    set_gerrit_config gitweb.type gitiles
+    su-exec ${GERRIT_USER} cp -f ${GERRIT_HOME}/gitiles.jar ${GERRIT_SITE}/plugins/gitiles.jar
+  else
+    set_gerrit_config gitweb.type gitweb
+    set_gerrit_config gitweb.cgi "/usr/share/gitweb/gitweb.cgi"
+  fi
```

Comment:
I wonder if we can make this more flexible like `set_gerrit_config gitweb.type "${GITWEB_TYPE}"` without any conditions because gitweb.type can be set to other values like cgit. 
Then we can do some conditional stuff according to the value of $GITWEB_TYPE such as copy jar file or setup gitweb.cgi.